invenio_records_resources/services/custom_fields/base.py

```diff
@@ -30,3 +30,7 @@ def mapping(self):
     def schema(self):
         """Marshmallow schema for vocabulary custom fields."""
         pass
+
+    def ui_schema(self):
```

Comment:
wondering if it should be `serialize` and `deserialize` instead of `schema` and `ui_schema`